libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss

```
@@ -9,7 +9,7 @@ $divider-max-width-breakpoint: utils.$page-content-max-width-default + (utils.si
   position: relative;
 
   @include utils.media('<medium') {
-    width: 100%;
+    inline-size: 100dvi;
```

Consider adding a small comment here: 
```suggestion
    // dynamic viewport width is used here to account for different scrollbar behaviors in browsers
    inline-size: 100dvi;
```